index.html

```diff
@@ -74,12 +83,14 @@ <h2>믿을 수 있는<br>
 					<a class="privacy" href="privacy.html">Privacy Policy</a>
 					<a class="faq" href="faq.html">FAQ</a>
 				</div>
-				<div class="sns">
-					<a href="https://www.facebook.com" target="_blank"><img src="./images/ic_facebook.png" alt="페이스북"></a>
-					<a href="https://www.twitter.com" target="_blank"><img src="./images/ic_twitter.png" alt="트위터"></a>
-					<a href="https://www.youtube.com" target="_blank"><img src="./images/ic_youtube.png" alt="유튜브"></a>
-					<a href="https://www.instagram.com" target="_blank"><img src="./images/ic_instagram.png" alt="인스타그램"></a>
-				</div>
+				<ul class="sns">
+          <li>
+            <a href="https://www.facebook.com" target="_blank"><img src="./images/ic_facebook.png" alt="페이스북"/></a>
+            <a href="https://www.twitter.com" target="_blank"><img src="./images/ic_twitter.png" alt="트위터"/></a>
+            <a href="https://www.youtube.com" target="_blank"><img src="./images/ic_youtube.png" alt="유튜브"/></a>
+            <a href="https://www.instagram.com" target="_blank"><img src="./images/ic_instagram.png" alt="인스타그램"/></a>
+          </li>

```

Comment:
❗️ 수정요청
ul은 unordered-list 라는 의미로 li는 해당 리스트의 아이템을 의미합니다. 리스트의 아이템이니 지금의 경우 각각의 a 태그를 li로 감싸주는 것이 더 적절합니다~

```suggestion
          <li><a href="https://www.facebook.com" target="_blank"><img src="./images/ic_facebook.png" alt="페이스북"/></a></li>
          <li><a href="https://www.twitter.com" target="_blank"><img src="./images/ic_twitter.png" alt="트위터"/></a></li>
          <li><a href="https://www.youtube.com" target="_blank"><img src="./images/ic_youtube.png" alt="유튜브"/></a></li>
          <li><a href="https://www.instagram.com" target="_blank"><img src="./images/ic_instagram.png" alt="인스타그램"/></a></li>
```